src/components/FilterFields/__snapshots__/DateRange.test.js.snap

```diff
@@ -24,97 +24,143 @@ exports[`<DateRange/> matches the snapshot with default props 1`] = `
       <Field
```

Comment:
This snapshot seems quite large. It's hard for me to judge if this change is ok or not.

Reply:
If you have suggestions on how to improve this, please let me know. The only way I know to reduce the size of a snapshot is to extract the component into smaller components and only produce snapshots for those. However, I'm only rendering 2 InputFields wrapped in a `div`, so there' not much room for improvement here as I see it.

Reply:
Yeah that's true. I think my feedback is a bit out of scope for this PR. Eventually maybe having functional tests instead of snapshots would make it easier for a reviewer who doesn't know exactly what the output should be. But that's out of scope for this change.

Reply:
Actually, there was much to improve on this snapshot 🤣 , it was a snapshot of a mounted component. Switched to a snapshot of a shallow mounted component in fc80f9f and thast's much smaller.